File: Java/JAVA/Pattern_8/12Sept/Pattern2.java

class Pattern2 {

	public static void main(String[] args) {

		for(int i=1;i<=4;i++) {
			int num=1;
			for(int j=3;j>=i;j--) {

				System.out.print("  ");
			}
			
			for(int k=1;k<=i;k++) {

				System.out.print(num+" ");
				num++;
			}
			System.out.println();
		}
	}
}

File: PPA/LogicBuilding/Harry/Pattern2.c


#include<stdio.h>

void main() {
	int n;
	scanf("%d",&n);

	for(int i=1; i<=n; i++) {
		int num= 0;
		for(int j=1; j<=n; j++) {

			if(j>=i) 
				printf("*");
			else
				printf(" ");
		}
		printf("\n");
	}
}
